### DISCUSSION TITLE
Add a 16bit parallel capacitor touch screen driver to the ESP32 s3 chip

### DISCUSSION THREAD

**Discussion by 774471872** [Bodmer/TFT_eSPI#2745] [New processor/display support or capability request]
Hello teachers

Could you please add a 16bit parallel display driver library to the esp32 s3 chip? The hardware is as follows: the display driver chip is st7796, and the touch driver chip is gt911. Thank you!

**Comment by pvetal** [comment]
Hi.
Are you solved problem?
I cant "friendship" to esp32 s3 with 4" display ST7796, very need any solution.
Thanks.
